terraform/cross-region-infra/dynamodb-global-tables.tf

```diff
@@ -0,0 +1,168 @@
+######################################################
+# Video global tables and replicas
+######################################################
+resource "aws_dynamodb_table" "videos_primary" {
+  # TODO: Got an error " A provider configuration reference must not be given in quotes" error when trying to reference like this 
+  # provider = "aws.${local.primary_region}"
+  provider = aws.eu-central-1
+
+  count = local.is_primary ? 1 : 0
```

Comment:
(note) I assume that we will deploy twice. Once in the primary `eu-central-1` and another in secondary `eu-west-1`.  

We will create DDB global table only when it's primary deployment.